src/touch/gestures.c

```diff
@@ -35,6 +35,8 @@
 // The minimum amount of sliding difference required, so that the gesture is detected as slide, not
 // touch
 static const uint8_t SLIDE_DETECTION_DIFF = MAX_SLIDER_POS * 0.04; // Percent of slider range
+static const uint8_t TAP_NO_SLIDE_DIFF = MAX_SLIDER_POS * 0.1; // Percent of slider range
```

Comment:
should update the comment above
- remove ", not touch"
- warn developers about potential future issues if both TAP and SLIDE gesture events are needed at the same time on a given slider (i.e. alternative pw entry UX concepts)

Reply:
i guess this was done such that slide gesture is still responsive enough. makes sense to me (i was worried about how changes could effect slide UX but this keeps it the same as before)

Reply:
nit: `TAP_NO_SLIDE_DIFF` --> `TAP_SLIDE_TOLERANCE`?

Reply:
Fixed.